### PR TITLE
magento/magento2#24564: Captcha not shown on contact form

### PR DESCRIPTION
**Pull request by jorge-armando**
- Fixed issue. Remove the <container> tag xml because in content textarea does not have how to render, and the module need this to apply the CAPTCHA.

### Description (*)

1. Removed the <container> tag in xml
2. Replaces $block->getChildHtml("form.additional.info") in to $block->getChildHtml()
3. Add in all pages the CAPTCHA block
4. All contact forms without initialized with XML file will be type contact.

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. Add a new contact form in admin, not in XML file.
2. Enter in the page and see if is rendered.

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by josefbehr** [comment]
@JorgeArmandoBittencourt please check and fix the failing tests. Why did you rename the block?

File: app/code/Magento/Captcha/view/frontend/layout/contact_index_index.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
-        <referenceContainer name="form.additional.info">
+        <referenceBlock name="contactForm">
```
Comment (Karlasa) [inline app/code/Magento/Captcha/view/frontend/layout/contact_index_index.xml:10]: name should not be removed and kept as container

File: app/code/Magento/Captcha/view/frontend/layout/default.xml [hunk 2]
```diff
@@ -7,6 +7,19 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
+        <referenceBlock name="contactForm">
```
Comment (Karlasa) [inline app/code/Magento/Captcha/view/frontend/layout/default.xml:10]: Why it's needed? You can't reference block that have not been created. 
cacheable="false" will disable FPC for all default layouts.

**Review by Karlasa — CHANGES_REQUESTED** [review]

**Comment by Karlasa** [comment]
@JorgeArmandoBittencourt  Your solution breaks FPC and is hard to understand meaning of why it's needed in first place. Could you improve Manual testing scenarios area because for me it's not clear what you mean by "Add a new contact form in admin". 

Cheers, 
Karla

**Comment by VladimirZaets** [comment]
@JorgeArmandoBittencourt , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**